### PR TITLE
adding first Databricks ecosystem guide - how to set up your project

### PR DESCRIPTION
**Pull request by asteichen**
## What are you changing in this pull request and why?

Adding first Databricks ecosystem guide, as initiated https://github.com/dbt-labs/docs.getdbt.com/issues/2598

This was done in collaboration with Databricks SMEs, Amy & Anders

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/sidebars.js [hunk 1]
```diff
@@ -843,6 +843,17 @@ const sidebarSettings = {
             "guides/dbt-ecosystem/adapter-development/7-verifying-a-new-adapter",
           ],
         },
+        {
+          type: "category",
+          label: "Databricks + dbt Guides",
```
Comment (amychen1776) [inline website/sidebars.js]: Since it's under guides, could we name this as Databricks + dbt?

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+## Setting up Service Principals
+
+When an analytics engineer runs a dbt project from their IDE, it is perfectly fine for the resulting queries to execute with that user’s identity. However, we want production runs to execute with a *service principal's* identity. As a reminder, a service principal is a headless account that does not belong to an actual person.
+
+Service principals are used to remove humans from deploying to production for convenience and security. Personal identities should not be used to build production pipelines because they could break if the user leaves the company or changes their credentials. Also, there should not be ad hoc commands modifying production data. Only scheduled jobs and running code that has passed CI tests and code reviews should be allowed to modify production data. If something breaks, there is an auditable trail of changes to find the root cause, can easily revert to the last working version of the code, and minimize the impact on end users.
+
+Let’s [create a service principal](https://docs.databricks.com/administration-guide/users-groups/service-principals.html#add-a-service-principal-to-your-databricks-account) in Databricks:
```
Comment (amychen1776) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]:
```suggestion
[Let’s create a service principal](https://docs.databricks.com/administration-guide/users-groups/service-principals.html#add-a-service-principal-to-your-databricks-account) in Databricks:
```

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+We will set up two different *catalogs* in Unity Catalog: **dev** and **prod**. A catalog is a top-level container for *schemas* (previously known as databases in Databricks), which in turn contain tables and views.
+
+Our dev catalog will be the development environment that analytics engineers interact with through their IDE. Developers should have their own sandbox to build and test objects in without worry of overwriting or dropping a coworker’s work; we recommend creating personal schemas for this purpose. In terms of permissions, they should only have access to the **dev** catalog.
+
+Only production runs will have access to data in the **prod** catalog. In a future guide, we will discuss a **test** catalog where our continuous integration/continuous deployment (CI/CD) system (such as GitHub Actions) can run `dbt test`.
+
+For now, let’s keep things simple and [create](https://docs.databricks.com/sql/language-manual/sql-ref-syntax-ddl-create-catalog.html) two catalogs either using the Data Explorer or in the SQL editor with these commands:
```
Comment (amychen1776) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]:
```suggestion
For now, let’s keep things simple and [create two catalogs](https://docs.databricks.com/sql/language-manual/sql-ref-syntax-ddl-create-catalog.html)  either using the Data Explorer or in the SQL editor with these commands:
```

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+## Setting up Databricks Compute
+
+When you run a dbt project, it generates SQL, which can run on All Purpose Clusters or SQL warehouses. We strongly recommend running dbt-generated SQL on a Databricks SQL warehouse. Since SQL warehouses are optimized for executing SQL queries, you can save on the cost with lower uptime needed for the cluster to run the queries. If you need to debug, you will also have access to a Query Profile. We recommend using a serverless cluster if you want to minimize the time spent on spinning up a cluster and removing the need to change cluster sizes depending on workflows.
+
+Let’s [create a Databricks SQL warehouse](https://docs.databricks.com/sql/admin/sql-endpoints.html#create-a-sql-warehouse):
+
+1. Click **SQL Warehouses** in the sidebar.
+2. Click Create SQL Warehouse.
```
Comment (amychen1776) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]:
```suggestion
2. Click *Create SQL Warehouse*.
```

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+3. Enter a name for the warehouse.
+4. Accept the default warehouse settings or edit them.
+5. Click Create
```
Comment (amychen1776) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]:
```suggestion
5. Click *Create*
```

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 6]
```diff
@@ -0,0 +1,203 @@
...
+**Schema-level grants:**
+
+|  | Source Data | Development catalog | Production catalog | Test catalog |
+| --- | --- | --- | --- | --- |
+| developers | use | use, create table & create view | use or none | none |
+| production service principal | use | none | use, create table & create view | none |
+| Test service principal | use | none | none | use, create table & create view |
+
+
+## Next steps
```
Comment (amychen1776) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md:100]: I would love to ask @joellabes and other folks on the product docs team if we need these next steps. Most of these are already linking to our docs so perhaps we don't actually need them?

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
+# How to set up your Databricks and dbt project
+
+Databricks and dbt Labs are partnering to help data teams think like software engineering teams and ship trusted data, faster. The dbt-databricks adapter enables both dbt users to leverage the latest Databricks features from their dbt project. Hundreds of customers are now using dbt and Databricks to build expressive and reliable data pipelines on the Lakehouse, generating data assets that enable analytics, ML, and AI use cases throughout the business.
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]:
```suggestion
Databricks and dbt Labs are partnering to help data teams think like software engineering teams and ship trusted data, faster. The dbt-databricks adapter enables dbt users to leverage the latest Databricks features in their dbt project. Hundreds of customers are now using dbt and Databricks to build expressive and reliable data pipelines on the Lakehouse, generating data assets that enable analytics, ML, and AI use cases throughout the business.
```

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+```sql
+create catalog if not exists dev;
+create catalog if not exists prod;
+```
+
+As long as your developer is given write access to the dev data catalog, there is no need to create the sandbox schemas ahead of time.
+
+## Setting up Service Principals
+
+When an analytics engineer runs a dbt project from their IDE, it is perfectly fine for the resulting queries to execute with that user’s identity. However, we want production runs to execute with a *service principal's* identity. As a reminder, a service principal is a headless account that does not belong to an actual person.
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md:28]: Love this extra context on why service principals are important ⭐

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+
+Service principals are used to remove humans from deploying to production for convenience and security. Personal identities should not be used to build production pipelines because they could break if the user leaves the company or changes their credentials. Also, there should not be ad hoc commands modifying production data. Only scheduled jobs and running code that has passed CI tests and code reviews should be allowed to modify production data. If something breaks, there is an auditable trail of changes to find the root cause, can easily revert to the last working version of the code, and minimize the impact on end users.
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]:
```suggestion
Service principals are used to remove humans from deploying to production for convenience and security. Personal identities should not be used to build production pipelines because they could break if the user leaves the company or changes their credentials. Also, there should not be ad hoc commands modifying production data. Only scheduled jobs and running code that has passed CI tests and code reviews should be allowed to modify production data. If something breaks, there is an auditable trail of changes to find the root cause, easily revert to the last working version of the code, and minimize the impact on end users.
```

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 10]
```diff
@@ -0,0 +1,145 @@
...
+
+We will set up two different *catalogs* in Unity Catalog: **dev** and **prod**. A catalog is a top-level container for *schemas* (previously known as databases in Databricks), which in turn contain tables and views.
+
+Our dev catalog will be the development environment that analytics engineers interact with through their IDE. Developers should have their own sandbox to build and test objects in without worry of overwriting or dropping a coworker’s work; we recommend creating personal schemas for this purpose. In terms of permissions, they should only have access to the **dev** catalog.
+
+Only production runs will have access to data in the **prod** catalog. In a future guide, we will discuss a **test** catalog where our continuous integration/continuous deployment (CI/CD) system (such as GitHub Actions) can run `dbt test`.
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]:
```suggestion
Only production runs will have access to data in the **prod** catalog. In a future guide, we will discuss a **test** catalog where our continuous integration/continuous deployment (CI/CD) system can run `dbt test`.
```

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 11]
```diff
@@ -0,0 +1,145 @@
...
+3. Click Generate.
+4. Copy the displayed token and click Done. (don’t lose it!)
+
+
+For your development credentials/profiles.yml:
+
+1. Set your default catalog to dev.
+2. Your developer schema should be named after yourself. We recommend dbt_<first_name_initial><last_name>.
+
+By doing this, dbt will create the developer schemas if it doesn’t already exist in the **dev** catalog but can be overridden to specify sources or production targets.
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: >but can be overridden to specify sources or production targets.

What does this mean in the context of making developer schemas?

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 12]
```diff
@@ -0,0 +1,145 @@
...
+Let’s set up our deployment environment:
+
+1. In Databricks, [set up your service principal’s token](https://docs.databricks.com/dev-tools/service-principals.html#use-curl-or-postman). To do this, you will need to open up your terminal.
+2. In your terminal, define 2 environment variables:
+    1. **DATABRICKS_HOST** - the url used to log into your Databricks workspace
+    2. **DATABRICKS_TOKEN** - your [personal access token](https://docs.databricks.com/dev-tools/auth.html#pat) (also used for dbt Cloud developer credentials)
+
+```bash
+export DATABRICKS_HOST="https://dbc-a1b2345c-d6e78.cloud.databricks.com"
+export DATABRICKS_TOKEN="dapi1234567890b2cd34ef5a67bc8de90fa12b"
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: I can imagine people copy-pasting these and failing. 

The dbt docs style guide describes how we do placeholders: https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#Placeholder-text 
```suggestion
export DATABRICKS_HOST="https://YOUR_UNIQUE_HOST.cloud.databricks.com"
export DATABRICKS_TOKEN="YOUR_PERSONAL_ACCESS_TOKEN"
```

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 13]
```diff
@@ -0,0 +1,145 @@
...
+```
+
+3. Next, run a curl command to get the “application_id” of your service principal. You can also get this value from the Databricks workspace admin console > Service Principals > Edit
+
+```bash
+curl -X GET \
+${DATABRICKS_HOST}/api/2.0/preview/scim/v2/ServicePrincipals \
+--header "Authorization: Bearer ${DATABRICKS_TOKEN}" \
+| jq .
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: Is `jq` installed by default? I thought it was a separate install. We shouldn't reference quality-of-life software that isn't guaranteed to be there

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+```
+
+4. Then you create a JSON file with the application ID retrieved from the above command, a comment to describe what the token will be used for, and the amount of time the token will be valid (in seconds) before it expires and will need to be regenerated.
+
+```bash
+vim create-service-principal.json
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: Likewise, let's not demand people use vim. We should just tell them to create a text file, and they can do it however they see fit (but we'll need to make clear that it has to be in the current working directory)

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+3. Next, run a curl command to get the “application_id” of your service principal. You can also get this value from the Databricks workspace admin console > Service Principals > Edit
+
+```bash
+curl -X GET \
+${DATABRICKS_HOST}/api/2.0/preview/scim/v2/ServicePrincipals \
+--header "Authorization: Bearer ${DATABRICKS_TOKEN}" \
+| jq .
+```
+
+4. Then you create a JSON file with the application ID retrieved from the above command, a comment to describe what the token will be used for, and the amount of time the token will be valid (in seconds) before it expires and will need to be regenerated.
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]:
```suggestion
4. Create a JSON file with the application ID retrieved from the above command, a comment to describe what the token will be used for, and the amount of time the token will be valid (in seconds) before it expires and will need to be regenerated.
```

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 16]
```diff
@@ -0,0 +1,145 @@
...
+
+```bash
+vim create-service-principal.json
+```
+
+```json
+{
+  "application_id": "<application-id>",
+  "comment": "<comment>",
+  "lifetime_seconds": 1209600
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: This is 2 weeks - what happens when this expires? Does someone have to go and do this from scratch every 2 weeks? Or is this token going to be a one-time use as we generate the _actual_ token we need?

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 17]
```diff
@@ -0,0 +1,145 @@
...
+4. Then you create a JSON file with the application ID retrieved from the above command, a comment to describe what the token will be used for, and the amount of time the token will be valid (in seconds) before it expires and will need to be regenerated.
+
+```bash
+vim create-service-principal.json
+```
+
+```json
+{
+  "application_id": "<application-id>",
+  "comment": "<comment>",
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]:
```suggestion
  "application_id": "<APPLICATION_ID_FROM_STEP_3>",
  "comment": "Service Principal Token for dbt deployment",
```

Please feel free to amend the example comment!

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md [hunk 18]
```diff
@@ -0,0 +1,145 @@
...
+
+5. Run one more curl command to get the service principal’s token:
+
+```bash
+curl -X POST \
+${DATABRICKS_HOST}/api/2.0/token-management/on-behalf-of/tokens \
+--header "Content-type: application/json" \
+--header "Authorization: Bearer ${DATABRICKS_TOKEN}" \
+--data @create-service-principal-token.json \
+| jq .
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: Again we can't assume that `jq` exists

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md (hunk 6 above)
Comment (joellabes) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md:100]: I don't mind it! A link to the best practices guide makes sense

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md (hunk 16 above)
Comment (joellabes) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: If it is meant to be a long-lived token, are we allowed to set its lifetime to 999999999999999? Is there any security implication to that? We should set our default to the actual best practice

**Review by joellabes — CHANGES_REQUESTED** [review]
Mostly looking good @asteichen! A handful of suggested tweaks, lmk if you disagree

File: website/docs/guides/dbt-ecosystem/databricks-guides/dbt-unity-catalog-best-practices.md [hunk 19]
```diff
@@ -0,0 +1,61 @@
...
+
+If you use multiple Databricks workspaces to isolate development from production, you can use dbt Cloud’s [environment variables](https://docs.getdbt.com/docs/build/environment-variables) in your connection config strings to reference multiple workspaces from one dbt Cloud project. You can also do the same thing for your SQL warehouse so you can have different sizes based on your environments.
+
+To do so, use dbt's [environment variable syntax](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-environment-variables#special-environment-variables) for Server Hostname of your Databricks workspace URL and HTTP Path for the SQL warehouse in your connection settings. Note that Server Hostname still needs to appear to be a valid domain name to pass validation checks, so you will need to hard-code the domain suffix on the URL, eg {{env_var('DBT_HOSTNAME')}}.cloud.databricks.com and the path prefix for your warehouses, eg /sql/1.0/warehouses/{{env_var('DBT_HTTP_PATH')}}.
```
Comment (joellabes) [inline website/docs/guides/dbt-ecosystem/databricks-guides/dbt-unity-catalog-best-practices.md]:
```suggestion
To do so, use dbt's [environment variable syntax](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-environment-variables#special-environment-variables) for Server Hostname of your Databricks workspace URL and HTTP Path for the SQL warehouse in your connection settings. Note that Server Hostname still needs to appear to be a valid domain name to pass validation checks, so you will need to hard-code the domain suffix on the URL, eg `{{env_var('DBT_HOSTNAME')}}.cloud.databricks.com` and the path prefix for your warehouses, eg `/sql/1.0/warehouses/{{env_var('DBT_HTTP_PATH')}}`.
```

**Review by joellabes — CHANGES_REQUESTED** [review]

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md (hunk 11 above)
Comment (asteichen) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: Looks like something might've been removed that would put that in a better context. 

This was meant to tell them that the **Catalog** they put as the default config can be overridden later (you're not stuck working in one Catalog). It doesn't have anything to do with dev **schemas**.


I think we could change it to:

> By doing this, dbt will create the developer schemas if it doesn’t already exist in the dev catalog. The default catalog in your project configuration can be overridden to specify sources or production targets in other Catalogs.

Comment (asteichen) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: @joellabes @amychen1776  does that edit make sense to you?

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md (hunk 16 above)
Comment (asteichen) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: That was the default number in Databrick's doc. I think when I tested in the UI the max was something around 3 years (might not be the same when doing it this way). 

I didn't want to change it in case Databricks selected it for security reasons. It's typically considered bad practice to have tokens like that that don't expire but realistically it would not be practical to rotate tokens for dbt Cloud service principles

Comment (asteichen) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: Personally, I would use the max value but I wasn't a security-minded developer :)

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md (hunk 18 above)
Comment (asteichen) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: This is straight out of the Databricks doc linked directly above it. There are details in there about alternatives. I think reproducing everything in the Databricks doc would make our guide hard to follow

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md (hunk 11 above)
Comment (joellabes) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: I like the proposed wording, but if it's got nothing to do with dev schemas or the development environment then it feels like it should move somewhere else? At minimum to a new paragraph. 

Also "by doing this" is slightly inaccurate isn't it? dbt will create developer schemas even if you don't name your schema after yourself, and I don't think it'll care what the default catalog is. I think you can just state it as a fact: 

During your first invocation of `dbt run`, dbt will create the developer schema if it doesn't already exist in the dev catalog.

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md (hunk 16 above)
Comment (joellabes) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: >when I tested in the UI

If this is possible in the UI, why are we messing around on the command line? I thought the docs said this _had_ to be done programmatically which is the only reason I didn't complain earlier 😅

Comment (joellabes) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: > Personally, I would use the max value 

same

>but I wasn't a security-minded developer :)

also same

Can we update it to 1 year, and tell people to set an annual calendar reminder to renew it? This feels super janky.

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md (hunk 18 above)
Comment (joellabes) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: >There are details in there about alternatives

Any code we provide is going to be copy-pasted and run immediately without any review, and for most people it's going to fail. The fact that there are instructions on a different website saying "by the way this won't work immediately" doesn't solve that IMO. 

Since this is a one-off process, I would make a pretty strong case for our section of the documentation being based on the Postman instructions, not the CLI (assuming that there's no UI - see above comment). Then we could even provide a Postman example that people could run themselves, right?

**Review by joellabes — CHANGES_REQUESTED** [review]

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md (hunk 16 above)
Comment (asteichen) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: > If this is possible in the UI, why are we messing around on the command line?

You can create them in the UI for users who can log into the workspace. Service principles cannot, so it needs to be done programmatically (at least for now, I hope they add it to the admin UI).

Comment (asteichen) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: > Can we update it to 1 year

Yeah I'll update it and add a note so they know it'll expire

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md (hunk 18 above)
Comment (asteichen) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: Removed these steps since there are too many edge cases to consider

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md (hunk 11 above)
Comment (asteichen) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: Updated!

File: website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md (hunk 16 above)
Comment (asteichen) [reply website/docs/guides/dbt-ecosystem/databricks-guides/how-to-set-up-your-databricks-dbt-project.md]: Removed these steps since there are too many edge cases to consider. Added a comment when directing them to the Databricks doc about the validity time

**Review by joellabes — APPROVED** [review]
love it! ty 🎉
